src/genie/libs/parser/iosxe/show_cts.py

```diff
@@ -0,0 +1,250 @@
+import re
+
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Optional
+
+
+# ==============================
+# Schema for:
+#  * 'show cts environment-data'
+# ==============================
+class ShowCtsEnvironmentDataSchema(MetaParser):
+    """Schema for show cts environment-data."""
+
+    schema = {
+        "cts_env": {
+          "current_state": str,
+          "last_status": str,
+          Optional("sgt_tags"): str,
+          Optional("tag_status"): str,
+          Optional("server_list_name"): str,
+          Optional("server_count"): int,
+          Optional("servers"): list
+            # {
+            #   Optional("server_ip"): str,
+            #   Optional("port"): int,
+            #   Optional("aid"): str,
+            #   Optional("server_status"): str,
+            #   Optional("auto_test"): str,
+            #   Optional("keywrap_enable"): str,
+            #   Optional("idle_time_mins"): int,
+            #   Optional("dead_time_secs"): int
+            # }
+          ,
+          Optional("security_groups"): list
+            # {
+            #   Optional("sec_group"): str,
+            #   Optional("sec_group_name"): str
+            # }
+          ,
+          Optional("env_data_lifetime_secs"): str,
+          Optional("last_update"): {
+            Optional("date"): str,
+            Optional("time"): str,
+            Optional("time_zone"): str
+          },
+          Optional("expiration"): str,
+          Optional("refresh"): str,
+          "state_machine_status": str,
+          Optional("retry_timer_status"): str
+        }
+    }
+
+
+
+# ==============================
+# Parser for:
+#  * 'show cts environment-data'
+# ==============================
+class ShowCtsEnvironmentData(ShowCtsEnvironmentDataSchema):
+    """Parser for show cts environment-data"""
+
+    cli_command = ['show cts environment-data']
+
+    def cli(self, output=None):
+        if output is None:
+            out = self.device.execute(self.cli_command[0])
+        else:
+            out = output
+
+        cts_env_dict = {}
+
+        current_state_capture = re.compile(r"^Current\s+state\s=\s+(?P<state>.*$)", re.MULTILINE)
```

Comment:
I don't think you need multiline.

Reply:
I meant you do not need multiline at all (for any of them)

Reply:
This has been updated